scripts/sandbox_start.sh

```diff
@@ -73,7 +73,7 @@ healthCheck()
 # for now we wait until nginx pick up the VIRTUAL_HOST
 sleep 3
 n=0
-maxAttempts=180
+maxAttempts=200
```

Comment:
Why this change

Reply:
for my local this time always work .. especially for demo resolver .. otherwise it fails once n work second time.. @aholovko also mentioned similar problem.